L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCConcentratorData.cc

```diff
@@ -0,0 +1,113 @@
+
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "DataFormats/L1THGCal/interface/HGCalConcentratorData.h"
+#include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
+#include "DataFormats/Common/interface/AssociationMap.h"
+#include "DataFormats/ForwardDetId/interface/HGCalTriggerDetId.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
+#include "L1Trigger/L1THGCalUtilities/interface/HGCalTriggerNtupleBase.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerTools.h"
+
+class HGCalTriggerNtupleHGCConcentratorData : public HGCalTriggerNtupleBase {
+public:
+  HGCalTriggerNtupleHGCConcentratorData(const edm::ParameterSet& conf);
+  ~HGCalTriggerNtupleHGCConcentratorData() override{};
+  void initialize(TTree&, const edm::ParameterSet&, edm::ConsumesCollector&&) final;
+  void fill(const edm::Event& e, const edm::EventSetup& es) final;
+
+private:
+  void clear() final;
+
+  HGCalTriggerTools triggerTools_;
+
+  edm::EDGetToken concentrator_data_token_;
+  edm::ESHandle<HGCalTriggerGeometryBase> geometry_;
+
+  static constexpr unsigned kSectorOffset_ = 7;
+  static constexpr unsigned kSectorMask_ = 0x7;
```

Comment:
I missed that from the last round of comments: these 2 constants are not needed anymore